cpg-language-go/src/test/kotlin/de/fraunhofer/aisec/cpg/frontends/golang/GoLanguageFrontendTest.kt

```diff
@@ -1017,17 +1017,27 @@ class GoLanguageFrontendTest : BaseTest() {
     fun testBuildTags() {
         val stdLib = Path.of("src", "test", "resources", "golang-std")
         val topLevel = Path.of("src", "test", "resources", "golang", "buildtags")
+
+        // make sure we parse main.go as the last
```

Comment:
Will the test fail if we change this ordering? Doesn't this suggest that we implemented some feature that is ultimately not reliably available?

Reply:
We always had this limitation, since we do not have a proper ordering of dependencies in the `SymbolResolver`. For Go, we somewhat minimise this problem because we have special functions that print out the package dependency tree if Go is installed and feed that as an input to the translation manager. But in the long run we should figure out the dependency tree based on our imports (that we have now). But this is probably work for an extra PR.